modules/ossm-deploying-bookinfo-application.adoc

```diff
@@ -24,12 +24,17 @@ The *Display name* and *Description* fields provide supplementary information an
 
 . Click *Create*.
 
-. Apply the the {Istio} injection label to the `bookinfo` namespace by entering the following command at the command line:
+. Apply the {Istio} injection label to the `bookinfo` namespace by entering the following command at the CLI:
 +
 [source,terminal]
 ----
-$ oc label namespace bookinfo istio.io/rev=<revision-name>
+$ oc label namespace bookinfo istio-injection=enabled
 ----
++
+[NOTE]
+====
+Note: In this example, the name of the `Istio` resource is `default`. If the `Istio` resource name is someting else, you must set the `istio.io/rev` label to the name of the `Istio` resource instead of adding the `istio-injection=enabled` label.
```

Comment:
```suggestion
In this example, the name of the `Istio` resource is `default`. If the `Istio` resource name is different, you must set the `istio.io/rev` label to the name of the `Istio` resource instead of adding the `istio-injection=enabled` label.
```

Reply:
Is 'Istio` the name of the resource? Do you need the `` around it.

Reply:
done